src/Chapter1.hs

```diff
@@ -620,7 +627,10 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = last2 $ abs n where
+  last2 x = last1 x + last1 (x `quot` 10)
+  last1 x = x `rem` 10
```

Comment:
For `last1` you could utilise the `lastDigit` function from the above 😉 

Also note, that to make your solution even shorter you could find the standard [`quotRem`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:quotRem) function handy, that actually combines inside both `quot` and `rem` in one go.

So you could write it this way:

```haskell
(first, second) = quoteRem x 10
```

You can see how we could pattern match on the pair 🙂